files/src/program_manager.py

```diff
@@ -44,8 +44,8 @@ def main(arguments):
         pid = Popen(["python3", program]).pid
         programs[program] = pid
 
-    if c_files or cpp_files:
-        print('Compiling source files...')
+    #if c_files or cpp_files:
```

Comment:
can you remove these changes to the program manager? this is a relic from when I was working on this branch... alternatively I can revert these.

Reply:
Yeah, you can do it I wouldn't mind